stackstorm-compose/README.md

```diff
@@ -0,0 +1,305 @@
+# StackStorm in Docker Compose
+
+This docker-compose is provided as a way to allow someone to "get up and running" quickly with StackStorm using Docker. It is not designed to be used in production, but rather a way to test out StackStorm and facilitate pack development.
+
+## TL;DR
+
+```shell
+git clone git@github.com:stackstorm/st2-dockerfiles
+cd st2-docker/stackstorm-compose
+docker-compose up -d
+docker-compose exec st2api st2ctl reload --register-all  # you should only have to do this the first time
+docker-compose exec st2client bash  # this gives you access to the st2 command line
```

Comment:
After "logging in" into st2client it prints K8s-relevant information related to `Helm values`, see https://github.com/StackStorm/st2-dockerfiles/blob/2f2b85497ed48491775654c4377502d793d8cbd5/base/files/.welcome.sh#L19

Can we modify Docker image to handle that case? For example, based on env variable presence like `KUBERNETES_PORT` which exists in K8s environment, but absent in Docker-compose env.

Reply:
IMHO images should not know if they're running under k8s or not. We should treat them like plain old Dockerfiles. This level of abstraction enables images to be run under docker-compose or k8s without any further development.

Reply:
@TunaYagci, I agree in general as this would add complexity and needs to be maintained. But I don't see an issue to add another if check to the script and check if that env variable is set. This should not prevent the image from being used in docker-compose, k8s or something else.

Or do I miss the point here?

Reply:
@winem if you mean that stackstorm 3.3 (or newest images) is going to be updated with this change on **everywhere**, that's okay. I'm saying that because I never built the images by myself and always downloaded them docker hub.